Cargo.toml

```diff
@@ -44,7 +44,7 @@ mime = "0.3"
 nanoid = "0.4"
 percent-encoding = "2"
 port_check = "0.1"
-qrcodegen = "1"
+qrcode = "0.12.0"
```

Comment:
By the way, there's also [fast_qr](https://github.com/erwanvivien/fast_qr) which might be better maintained and it has fewer dependencies. Maybe worth checking out?

Reply:
Sure, I'll try it.